cmd/notation/key.go

```diff
@@ -48,6 +48,23 @@ func keyCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "key",
 		Short: "Manage keys used for signing",
+		Long: `Manage keys used for signing
+
+Example - Add a key to signing key list:
+  notation key add --plugin <plugin_name> --id <key_id> <key_name>
+
+Example - List keys used for signing:
+  notation key ls
+
+Example - Update key in signing key list:
+  notation key set [flags] <key_name>
+
+Example - Update the default signing key:
+  notation key update --default <key_name>
+
+Example - Delete the key from signing key list:
+  notation key delete [flags] <key_name>...
```

Comment:
wrong command line example

Reply:
No flags for delete command,

Reply:
@yizha1 `Set` is an alias of `Update`. Let's keep using the short alias in each command line example. It will be easy to type.

Reply:
Removed [flags]